akka-docs/src/main/paradox/scala/guide/tutorial_1.md

```diff
@@ -1,142 +1,86 @@
-upper layer of three actors (the root guardian, the user guardian, and the system guardian), thereafter the user created
-top-level actors (those directly living under `/user`) and the children of those. We now understand what the hierarchy
-looks like, but there are some nagging unanswered questions: _Why do we need this hierarchy? What is it used for?_
+### The actor lifecycle
+Actors pop into existence when created, then later, at user requests, they are stopped. Whenever an actor is stopped, all of its children are _recursively stopped_ too.
+This behavior greatly simplifies resource cleanup and helps avoid resource leaks such as those caused by open sockets and files. In fact, a commonly overlooked difficulty when dealing with low-level multi-threaded code is the lifecycle management of various concurrent resources.
 
-The first use of the hierarchy is to manage the lifecycle of actors. Actors pop into existence when created, then later,
-at user requests, they are stopped. Whenever an actor is stopped, all of its children are _recursively stopped_ too.
-This is a very useful property and greatly simplifies cleaning up resources and avoiding resource leaks (like open
-sockets files, etc.). In fact, one of the overlooked difficulties when dealing with low-level multi-threaded code is
-the lifecycle management of various concurrent resources.
+To stop an actor, the recommended pattern is to call `context.stop(self)` inside the actor to stop itself, usually as a response to some user defined stop message or when the actor is done with its job. Stopping another actor is also possible by calling `context.stop(actorRef)`. **It is considered a bad practice to stop arbitrary actors this way**, try sending them a `PoisonPill` or custom stop message
+instead.
 
-Stopping an actor can be done by calling `context.stop(actorRef)`. **It is considered a bad practice to stop arbitrary
-actors this way**. The recommended pattern is to call `context.stop(self)` inside an actor to stop itself, usually as
-a response to some user defined stop message or when the actor is done with its job.
-
-The actor API exposes many lifecycle hooks that the actor implementation can override. The most commonly used are
+The Akka actor API exposes many lifecycle hooks that you can override in an actor implementation. The most commonly used are
 `preStart()` and `postStop()`.
 
  * `preStart()` is invoked after the actor has started but before it processes its first message.
  * `postStop()` is invoked just before the actor stops. No messages are processed after this point.
 
-Again, we can try out all this with a simple experiment:
+Let's use the `preStart()` and `postStop()` lifecycle hooks in a simple experiment to observe the behavior when we stop an actor. Add the code shown in the snippet to your `ActorQuickStart` source file as follows:
+
+1. Add the class below the existing actor class definitions.
```

Comment:
Went through this section again and split things out/clarified, should flow better now (without repeating ourselves too much).